boards/arm/tiva/lm3s6965-ek/configs/qemu-flat/defconfig

```diff
@@ -0,0 +1,109 @@
+#
+# This file is autogenerated: PLEASE DO NOT EDIT IT.
+#
+# You can use "make menuconfig" to make any modifications to the installed .config file.
+# You can then do "make savedefconfig" to generate a new defconfig file that includes your
+# modifications.
+#
+# CONFIG_FS_PROCFS_EXCLUDE_ENVIRON is not set
+# CONFIG_NSH_DISABLE_DATE is not set
+CONFIG_ARCH="arm"
+CONFIG_ARCH_BOARD="lm3s6965-ek"
+CONFIG_ARCH_BOARD_LM3S6965EK=y
+CONFIG_ARCH_CHIP="tiva"
+CONFIG_ARCH_CHIP_LM3S6965=y
+CONFIG_ARCH_CHIP_LM3S=y
+CONFIG_ARCH_CHIP_LM=y
+CONFIG_ARCH_STACKDUMP=y
+CONFIG_ARM_SEMIHOSTING_HOSTFS=y
+CONFIG_BOARDCTL_APP_SYMTAB=y
+CONFIG_BOARD_LOOPSPERMSEC=4531
+CONFIG_BUILTIN=y
+CONFIG_CLOCK_MONOTONIC=y
+CONFIG_DEBUG_ASSERTIONS=y
+CONFIG_DEBUG_ERROR=y
+CONFIG_DEBUG_FEATURES=y
+CONFIG_DEBUG_FULLOPT=y
+CONFIG_DEBUG_INFO=y
+CONFIG_DEBUG_SYMBOLS=y
+CONFIG_DEBUG_WARN=y
+CONFIG_ELF=y
+CONFIG_EXAMPLES_ELF=y
+CONFIG_EXAMPLES_ELF_DEVPATH="/dev/ram5"
+CONFIG_EXAMPLES_HELLO=y
+CONFIG_EXAMPLES_MODULE=y
+CONFIG_EXAMPLES_MODULE_DEVPATH="/dev/ram6"
+CONFIG_EXAMPLES_NETTEST=m
+CONFIG_EXAMPLES_WGET=y
+CONFIG_FS_BINFS=y
+CONFIG_FS_HOSTFS=y
+CONFIG_FS_PROCFS=y
+CONFIG_FS_TMPFS=y
+CONFIG_HOST_MACOS=y
+CONFIG_LIBC_DLFCN=y
+CONFIG_LIB_ENVPATH=y
+CONFIG_MAX_TASKS=16
+CONFIG_MMCSD=y
+CONFIG_MMCSD_SPICLOCK=12500000
+CONFIG_NET=y
+CONFIG_NETDB_DNSCLIENT=y
+CONFIG_NETDB_DNSCLIENT_NAMESIZE=64
+CONFIG_NETDB_DNSSERVER_NOADDR=y
+CONFIG_NETDB_MAX_IPADDR=2
+CONFIG_NETINIT_DHCPC=y
+CONFIG_NETINIT_NOMAC=y
+CONFIG_NETUTILS_NETCAT=y
+CONFIG_NETUTILS_TELNETD=y
+CONFIG_NETUTILS_TFTPC=y
+CONFIG_NETUTILS_WEBCLIENT=y
+CONFIG_NET_BROADCAST=y
+CONFIG_NET_ICMP=y
+CONFIG_NET_ICMP_SOCKET=y
+CONFIG_NET_MAX_LISTENPORTS=8
+CONFIG_NET_STATISTICS=y
+CONFIG_NET_TCP=y
+CONFIG_NET_UDP=y
+CONFIG_NET_UDP_CHECKSUMS=y
+CONFIG_NFILE_DESCRIPTORS=8
+CONFIG_NSH_ARCHINIT=y
+CONFIG_NSH_BUILTIN_APPS=y
+CONFIG_NSH_DISABLE_IFUPDOWN=y
+CONFIG_NSH_FILEIOSIZE=512
+CONFIG_NSH_FILE_APPS=y
+CONFIG_NSH_LINELEN=1000
```

Comment:
hi @yamt, could you help explain why `CONFIG_NSH_LINELEN` is set to 1000 please?

Reply:
the default was too small for me.
i don't remember why i chose this particular value though.
it seems a bit too large. :-)